scrapers/Filename.py

```diff
@@ -0,0 +1,31 @@
+import json
+import os
+import sys
+
+from py_common import graphql
+from py_common import log
```

Comment:
we use something like this to be more user friendly
```suggestion
try:
    from py_common import graphql
    from py_common import log
except ModuleNotFoundError:
    print(
        "You need to download the folder 'py_common' from the community repo! (CommunityScrapers/tree/master/scrapers/py_common)",
        file=sys.stderr)
    sys.exit()
```